langfuse/task_manager.py

```diff
@@ -146,16 +146,19 @@ def init_resources(self):
             consumer.start()
             self._consumers.append(consumer)
 
-    def add_task(self, event):
+    def add_task(self, event: dict):
         try:
-            self._log.debug("Adding task")
+            self._log.debug(f"adding task {event}")
+            json.dumps(event, cls=EventSerializer)
```

Comment:
if event is not serializable, it will log an error to the console without causing an exception.